chalicelib/action.py

```diff
@@ -99,11 +99,8 @@ def _list_action(self):
 
         Supported arguments:
         "today" - List the event for the todays date
-        "date" - The date as a string. Use ":" as delimiter for two dates: "2019-01-01:2019-01-02"
+        "date" - The date as a string.
```

Comment:
Would be good to add what a date as string might look like. 
`"date" - Tje date as a string. 2019-01`

Reply:
The reason for removing the example is that we have example in the function `get_list_data` instead.

I think the best way to show good examples would be in `perform_action`-methods docstring  (which is the data displayed when you run `/timereport help`)